From 10def67d03ac5ae5cdd4a9b82f643869deb43f60 Mon Sep 17 00:00:00 2001
From: allgdante <allan.garret@gmail.com>
Date: Thu, 5 Jul 2018 17:32:46 +0200
Subject: [PATCH 11/11] utils: adjust rc functions for aa-remove-unknown

Update the path to the rc.apparmor.functions file to the one we can find
inside Alpine.
No intended functional changes
---
 utils/aa-remove-unknown | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/aa-remove-unknown b/utils/aa-remove-unknown
index d3bd9144..0b9ead7d 100644
--- a/utils/aa-remove-unknown
+++ b/utils/aa-remove-unknown
@@ -15,7 +15,7 @@
 #    along with this program. If not, see <http://www.gnu.org/licenses/>.
 # ----------------------------------------------------------------------
 
-APPARMOR_FUNCTIONS=/lib/apparmor/rc.apparmor.functions
+APPARMOR_FUNCTIONS=/usr/libexec/apparmor/rc.apparmor.functions
 APPARMORFS=/sys/kernel/security/apparmor
 PROFILES="${APPARMORFS}/profiles"
 REMOVE="${APPARMORFS}/.remove"
-- 
2.17.1

